.github/workflows/releases.yml

```diff
@@ -25,29 +27,38 @@ jobs:
         env:
           GH_OAUTH_CLIENT_ID: 178c6fc778ccc68e1d6a
           GH_OAUTH_CLIENT_SECRET: ${{secrets.OAUTH_CLIENT_SECRET}}
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          GITHUB_TOKEN: ${{secrets.UPLOAD_GITHUB_TOKEN}}
   msi:
     needs: goreleaser
     runs-on: windows-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
-      - name: Set up Go 1.13
-        uses: actions/setup-go@v1
-        with:
-          go-version: 1.13
+        uses: actions/checkout@v2
       - name: Download gh.exe
-        uses: ./.github/actions/download-exe
         id: download_exe
+        shell: bash
+        run: |
+          curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
+          bin/hub release download "${GITHUB_REF#refs/tags/}" -i '*windows_amd64*.zip'
```

Comment:
Now that we're uploading `.zip` instead of `.exe` to our relases, I found myself struggling to update the `download-exe` action accordingly. Instead, I've opted to use hub to make downloading and extracting simpler.